monai/visualize/class_activation_maps.py

```diff
@@ -12,15 +12,30 @@
 import warnings
 from typing import Callable, Dict, Sequence, Union
 
+import numpy as np
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 
 from monai.networks.utils import eval_mode, train_mode
+from monai.transforms import ScaleIntensity
 from monai.utils import ensure_tuple
-from monai.visualize import default_normalizer, default_upsampler
+from monai.visualize import default_upsampler
```

Comment:
we could address the isort issue by
```suggestion
from monai.visualize.visualizer import default_upsampler
```